README.md

```diff
@@ -113,6 +113,26 @@ or specifying it in the configuration file
 params.validationLenientMode = true
 ```
 
+### Formats
+
+Formats can be used to check `string` values for certain properties.
+
+Following table shows all additional formats implemented in this plugin. These formats will also be converted to the correct type.
+
+| Format | Description |
+|-----------|-------------|
+| file-path | Automatically checks if the file exists and transforms the `String` type to a `Nextflow.File` type, which is usable in Nextflow processes as a `path` input |
+| directory-path | Automatically checks if the directory exists and transforms the `String` type to a `Nextflow.File` type, which is usable in Nextflow processes as a `path` input. This is currently synonymous for `file-path`. |
```

Comment:
This is no longer correct (I added more types), you can check the samplesheet docs for this